andoriyaprashant/DevDocsHub
# fix-bug-dark-mode-toogle

**Description:**
This PR addresses the issue where the "Dark Mode" button successfully toggled the theme, but the button text did not change accordingly. The text remained "Dark Mode" even after switching to dark mode, which could confuse users.

**Issue:**
Fixes #48

**Problem:**
When the user clicked on the "Dark Mode" button, the UI theme switched correctly, but the button label did not update. Instead, it continued to display "Dark Mode" even though dark mode was already active. The button text should have switched to "Light Mode" after enabling dark mode.

**Fix:**
I updated the logic behind the button so that the text now dynamically changes based on the current theme mode.
When dark mode is enabled, the button text changes to "Light Mode," and when light mode is active, the button text correctly displays "Dark Mode."

**Steps to test the fix:**
Click the "Dark Mode" button.
Verify that the UI theme switches to dark mode.
Check that the button text changes to "Light Mode" once dark mode is enabled.
Click the button again to switch back to light mode and confirm that the button text updates to "Dark Mode" accordingly.

**Screenshots:**
Before:

![image](https://github.com/user-attachments/assets/49e967dd-b659-41c6-b2b5-d438b1602a70)

After:
![image](https://github.com/user-attachments/assets/e58d311f-b80f-4506-bcff-b1ebe516dc2a)
